src/ProjectTemplates/Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/package.json

```diff
@@ -6,6 +6,9 @@
     "bootstrap": "^4.1.3",
     "jquery": "3.3.1",
     "merge": "^1.2.1",
+////#if (IndividualLocalAuth)
+//    "oidc-client": "^1.6.1",
+////#endif
```

Comment:
If you're changing any `package.json` files, you also need to regenerate the `package-lock.json` file. Otherwise `npm ci` will fail with:

```
npm ERR! cipm can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
```

Reply:
This probably applies to the Angular project too.

Reply:
I'm not sure how practical it will be to have any conditionals in `package.json`, because then you'd need some potentially large number of corresponding conditionals in `package-lock.json`.

The safer option is probably just to include the `oidc-client` dependency in all cases, regardless of the `IndividualLocalAuth` flag.